src/Components/Nav/Nav.jsx

```diff
@@ -6,47 +6,56 @@ import './Nav.scss';
 class Nav extends Component {
   constructor() {
     super();
-    this.state = {};
+    this.state = { isSticky: false, scrollY: 0 };
+    this.ref = React.createRef();
   }
 
+  componentDidMount() {
+    window.addEventListener('scroll', this.scroll);
+  }
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.scroll);
+  }
+  scroll = () => {
+    if (window.scrollY > 10) {
+      this.setState({
+        isSticky: true,
+      });
+    }
+    if (window.scrollY <= 10) {
+      this.setState({
+        isSticky: false,
+      });
+    }
+  };
```

Comment:
```suggestion
  scroll = () => {
      this.setState({
        isSticky: window.scrollY > 10,
      });
    }
  };
```
- scroll 함수명 수정해주세요.
- boolean 값을 좀 더 효율적으로 사용할 수 있겠네요!

Reply:
네 수정하겠습니다!